cheddar/cheddar-server/src/main/java/com/clicktravel/cheddar/server/http/filter/logging/LoggingRequestFilter.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Cheddar is currently on Jersey 2.22.2. Starting with Jersey 2.23 there is a new logging implementation available. It may be more appropriate to use this rather than implement or own : https://jersey.java.net/documentation/latest/logging_chapter.html
If we were to adopt this, we still require this log output to form part of the new log this pull request is implementing.